### ISSUE TITLE
Creating a service with 1.0.0-alpha.2 completely ignores the --name parameter/option and uses "aws-nodejs" instead 

### ISSUE THREAD

**Issue by codepreneur** [serverless/serverless#1616]
##### Serverless Framework Version: 

```
1.0.0-alpha.2
```

##### Operating System: 

![image](https://cloud.githubusercontent.com/assets/3421858/16957931/5fe5d11e-4dd6-11e6-9d95-a5734cdee160.png)

##### Expected Behavior:

To not ignore `--name` parameter and name the service accordingly

##### Actual Behavior:

![image](https://cloud.githubusercontent.com/assets/3421858/16959413/e52e48c8-4ddc-11e6-8a24-bca3a7384989.png)

and

![image](https://cloud.githubusercontent.com/assets/3421858/16959427/0180322a-4ddd-11e6-9446-e8fe6e2ebcbc.png)

**Comment by eahefnawy** [comment]
We removed the `--name` option completely. You can now only generate a template (in the current directory), and then name the service/template whatever you like in the `service` property of `serverless.yaml`. We think this makes using the CLI much easier. What you think about this change?

**Comment by codepreneur** [comment]
I think removing the `--name` option is bad user experience unless there is a message somewhere telling the user that the service has been named "aws-nodejs" by default and the user should rename it before deploying.

I think the best solution is to keep `--name` option, but have some service name by default if the option is not specified.

**Comment by flomotlik** [comment]
Yup agreed @codepreneur . Basic reason we removed it is because it lead to issues with our templating system and is more work that needs to be done before it can be back in. We'll get it back in the future.

**Comment by flomotlik** [comment]
Added a new task in #1902 that covers this, so closing this one.
